app/models/adviser.rb

```diff
@@ -15,12 +15,12 @@ class Adviser < ActiveRecord::Base
   validates :travel_distance,
     presence: true,
     inclusion: { in: TravelDistance.all },
-    unless: :covers_whole_of_uk?
+    if: -> { covers_whole_of_uk === false }
```

Comment:
I changed these lines back to the regular predicate and the specs are all green. Either the specs aren't covering this case well enough, or we don't need threequals for this comparison after all? 

Also one thing to note - their is a spec https://github.com/moneyadviceservice/rad/blob/5659-hidden-error-messages/spec/models/adviser_spec.rb#L40 that covers those validations and whether they fire given certain values for `covers_whole_of_uk`, we should beef that up to cover the case of `nil`, `false` and `true` now we essentially have the tri-state boolean.